cli/medperf/tests/commands/dataset/test_create.py

```diff
@@ -94,10 +93,8 @@ def exists(path):
         else:
             spy.asset_not_called()
 
-    @pytest.mark.parametrize("cube_uid", rand_l(1, 5000, 5))
-    def test_get_prep_cube_gets_prep_cube_if_provided(
-        self, mocker, preparation, cube_uid, comms, ui
-    ):
+    @pytest.mark.parametrize("cube_uid", [1776, 4342, 573])
+    def test_get_prep_cube_gets_benchmark_cube(self, mocker, preparation, cube_uid, comms, ui):
```

Comment:
The test name here seems to be changed by mistake, right?

Reply:
You're right! I missed that one. This was most probably due to merge conflicts and didn't realize the other name was the correct one